pkgs/development/tools/documentation/doxyrest/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchgit, cmake, lib, expat, ragelDev, which, lua5 }:
+
+stdenv.mkDerivation rec {
+  name = "doxyrest";
+  # Somehow fetchFromGithub was ignoring submodules
+  src = fetchgit {
+    url = "https://github.com/vovkos/doxyrest_b";
+    rev = "3500bc0101c5ebe4921d8cca49b42e209e761971";
+    sha256 = "1h1r0g9yklgf2f8qv4dw0lnsldpcr5rch14q934fvxqbpjh9b7fn";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ expat lua5 ragelDev which cmake ];
```

Comment:
What does expat and lua5 are used for? I'm asking because I'm rather sure they belong to `buildInputs`.

Reply:
I'm not sure, `lua` is used for the configuration of the `doxyrest` binary, and `expat` is also used by the binary to read in XML files. Should they then belong in `buildInputs`?